connect/src/spirc.rs

```diff
@@ -1534,7 +1534,7 @@ struct CommandSender<'a> {
 }
 
 impl<'a> CommandSender<'a> {
-    fn new(spirc: &'a mut SpircTask, cmd: MessageType) -> CommandSender<'_> {
+    fn new(spirc: &'a mut SpircTask, cmd: MessageType) -> CommandSender<'a> {
```

Comment:
```suggestion
    fn new(spirc: &'a mut SpircTask, cmd: MessageType) -> Self {
```

it seems that it's even possible to turn all of the return type annotations in this `impl` block into `Self` (at least it works locally for me with rustc 1.81.0).

Reply:
Thanks. Also works with 1.74.0

Reply:
Well spotted.